tests/repository_test.py

```diff
@@ -82,8 +82,23 @@ def _test_hook_repo(
 def test_conda_hook(tempdir_factory, store):
     _test_hook_repo(
         tempdir_factory, store, 'conda_hooks_repo',
-        'foo', [os.devnull],
-        b'Test ' + five.to_bytes(os.devnull) + b'\n',
+        'sys-exec', [os.devnull],
+        b'conda-default\n',
+    )
+
+
+def test_conda_with_additional_dependencies_hook(tempdir_factory, store):
+    _test_hook_repo(
+        tempdir_factory, store, 'conda_hooks_repo',
+        'additional-deps', [os.devnull],
+        b'OK\n',
+        config_kwargs={
+            'hooks': [{
+                'id': 'additional-deps',
+                'args': ['-c', 'import requests; print("OK")'],
+                'additional_dependencies': ['requests'],
```

Comment:
let's use something that isn't requests, mccabe is nice and small and no external deps for example

Reply:
Done.